install/install.go

```diff
@@ -1051,6 +1405,68 @@ func K8sUninstaller(c *k8s.Client, o Options) error {
 	return nil
 }
 
+// K8sUninstaller for karmor uninstall
+func K8sUninstaller(c *k8s.Client, o Options) error {
+	var ns string
+
+	if o.Namespace != "" {
+		pods, _ := c.K8sClientset.CoreV1().Pods(o.Namespace).List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase==Running"})
+		if len(pods.Items) == 0 {
+			fmt.Println("❌\tKubeArmor is either not installed or you have specified incorrect namespace.\nℹ️\tPlease ensure you have installed KubeArmor or have specificed the correct namespace.")
+			return nil
+		}
+		ns = o.Namespace
+	} else {
+		if pods, _ := c.K8sClientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "kubearmor-app", FieldSelector: "status.phase==Running"}); len(pods.Items) == 0 {
+			fmt.Println("❌\tKubeArmor is not installed.\nℹ️\tPlease ensure you have installed KubeArmor.")
+			return nil
+		} else {
+			if len(pods.Items) > 0 {
+				ns = pods.Items[0].Namespace
+			}
+		}
+	}
+
+	settings := cli.New()
+	settings.SetNamespace(ns)
+	settings.RESTClientGetter()
+
+	actionConfig := new(action.Configuration)
+	if err := actionConfig.Init(settings.RESTClientGetter(), ns, os.Getenv("HELM_DRIVER"), log.Printf); err != nil {
+		log.Printf("%+v", err)
+		os.Exit(1)
```

Comment:
Why not return error?

Reply:
Oops, Already have a function to handle this, will be addressed in the commit. 👍